### PR TITLE
Ad 2782: ignore tracking fields if empty

### PR DESCRIPTION
**Pull request by smoreelAdhese**

### CODE REVIEW

**Comment by paulvv** [comment]
ok

**Comment by SolveSoul** [comment]
Ook OK van Endare 👍 

(mits kleine opmerking, zie hieronder)

File: sdk/build.gradle [hunk 1]
```diff
@@ -49,13 +49,11 @@ dependencies {
...
     androidTestImplementation 'androidx.test.ext:junit:1.1.0'
     androidTestImplementation 'androidx.test.espresso:espresso-core:3.2.0'
 }
 
 repositories {
     mavenCentral()
-}
-
-apply from: 'https://raw.githubusercontent.com/nisrulz/JCenter/master/installv1.gradle'
-apply from: 'https://raw.githubusercontent.com/nisrulz/JCenter/master/bintrayv1.gradle'
```
Comment (SolveSoul) [inline sdk/build.gradle:61]: Is er een reden waarom deze verwijderd is? Momenteel kan ik hierdoor niet publishen (kan het wel manueel terug toevoegen, maar dat is omslachtiger)

Comment (smoreelAdhese) [reply sdk/build.gradle:61]: In our opinion this was not secure. This downloads code directly from nisrulz github. If he changed his repo our build would change as wel. If you really need them you could make it secure by adding those files to our repo, that way we are in control.

Comment (SolveSoul) [reply sdk/build.gradle:61]: Fair enough

Comment (SolveSoul) [reply sdk/build.gradle:61]: Copied over the files to the repo for full control and ownership. See: 10ef83a8adc569d642b009e254d4119ef0e0fed8

**Pull request merged**